nipype/interfaces/dipy/base.py

```diff
@@ -100,7 +100,7 @@ def convert_to_traits_type(dipy_type, is_file=False):
     elif "string" in dipy_type and not is_file:
         return traits.Str, is_mandatory
```

Comment:
While we're switching from `traits.File`:

```suggestion
        return Str, is_mandatory
```

This will need an import.

@skoudoro Could you verify that this is fine and that you didn't explicitly intend to use `traits.Str` over `nipype.interfaces.base.Str` for some reason?

Reply:
I just check it and everything is fine.

Reply:
@oesteban Did we not want to change this?